### DISCUSSION TITLE
[@types/yeoman-environment] is not in sync with version of yeoman-environment package

### DISCUSSION THREAD

**Discussion by blackholegalaxy** [DefinitelyTyped/DefinitelyTyped#65640] [Issues with a @types package]
@types/yeoman-environment is still in v2 even if yeoman-environment bumped a long time ago to 3.x.y. This causes errors while using yeoman-generator since the main package requires @types/yeoman-environment with "*" but yeoman-environment at version 3. It then complain that the package is still at v2, seeing only typing version and not main package one.

[a comment by an automated account is not shown]

**Comment by neurolag** [comment]
When the latest version of `@types/`-packages don't match with the latest version of its respective package, this usually means one of the following:
  - The version didn't introduce any notable changes to its types
  - Noone cared to bring the type declarations up to date so far

I assume, the first one is the case here.

The version of the type declaration packages are usually specified as permissive as possible (thus the `"*"`) as they don't matter for the functionality of the code but rather for TypeScript's type checking.

Could you please provide a repository which shows the error you're facing?
